src/frontend/screens/Game/GamePage/index.css

```diff
@@ -147,10 +151,9 @@
     }
   }
 
-  //mainInfoWrapper
   .mainInfoWrapper {
     .mainInfo {
-      padding: max(25vh, 125px) min(4vw, 70px) min(5vh, 25px);
+      padding: 0px min(4vw, 70px);
```

Comment:
I'm using margin on the children instead of padding on the wrapper to have more variable control